### PR TITLE
Added getting [windowHandle] and [contentHandle].

### PR DESCRIPTION
**Pull request by Rsedaikin**

### CODE REVIEW

File: skiko/src/jvmMain/cpp/windows/drawlayer.cc [hunk 1]
```diff
@@ -13,6 +14,12 @@ extern "C"
     }
 
     JNIEXPORT jlong JNICALL Java_org_jetbrains_skiko_HardwareLayer_getWindowHandle(JNIEnv *env, jobject canvas, jlong platformInfoPtr)
+    {
+        JAWT_Win32DrawingSurfaceInfo* dsi_win = fromJavaPointer<JAWT_Win32DrawingSurfaceInfo *>(platformInfoPtr);
+        return (jlong) GetAncestor(dsi_win->hwnd, GA_PARENT);
```
Comment (olonho) [inline skiko/src/jvmMain/cpp/windows/drawlayer.cc:19]: Shalln't we assert that it's not NULL?

File: skiko/src/jvmMain/cpp/windows/drawlayer.cc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #define WIN32_LEAN_AND_MEAN
 #include <jawt_md.h>
+#include <Windows.h>
```
Comment (olonho) [inline skiko/src/jvmMain/cpp/windows/drawlayer.cc:3]: Usually it's `windows.h`, isn't it?

**Review by olonho — APPROVED** [review]
Looks good.

File: skiko/src/jvmMain/cpp/windows/drawlayer.cc (hunk 2 above)
Comment (Rsedaikin) [reply skiko/src/jvmMain/cpp/windows/drawlayer.cc:3]: Maybe, but on my PC really [Windows.h]

File: skiko/src/jvmMain/cpp/windows/drawlayer.cc (hunk 1 above)
Comment (Rsedaikin) [reply skiko/src/jvmMain/cpp/windows/drawlayer.cc]: Done.

**Pull request merged**